nservicebus/operations/auditing.md

```diff
@@ -30,7 +30,9 @@ The queue name to forward audit messages to
  
 ### OverrideTimeToBeReceived
 
-By default messages are forwarded to the audit queue with the [TimeToBeReceived](/nservicebus/messaging/discard-old-messages.md) of the original message. If a different TimeToBeReceived is required it have be achieved by setting OverrideTimeToBeReceived. This setting takes a [TimeSpan](https://msdn.microsoft.com/en-us/library/system.timespan.aspx).
+Version 5 and below will default TimeToBeReceived on the message forwarded to the audit queue to the [TimeToBeReceived](/nservicebus/messaging/discard-old-messages.md) of the original message. Starting with V6 no TimeToBeReceived will be set by default.
+
+You can force a TimeToBeReceived on audit messages by setting `OverrideTimeToBeReceived` using the configuration syntax below. 
 
 Note that while the phrasing is "forwarding a message" in the implementation it is actually "cloning and sending a new message". This is important when considering TimeToBeReceived since the time taken to receive and process the original message is not part of the TimeToBeReceived of the new audit message. So in effect the audit message receives the full time allotment of whatever TimeToBeReceived is used. 
```

Comment:
Should the note mention also if TTBR on original message is short and Audit queue is unavailable for longer than that time, audit message will be discarded?